package.json

```diff
@@ -58,14 +58,14 @@
     "react-jsonschema-form": "^1.6.1",
     "react-mailchimp-subscribe": "^2.1.0",
     "react-measure": "^2.1.2",
-    "react-native-rss-parser": "^1.3.0",
     "react-plotly.js": "^2.3.0",
     "react-router-dom": "^4.3.1",
     "react-scripts": "^3.0.1",
     "react-test-renderer": "^16.4.2",
     "react-tooltip": "^3.9.2",
     "react-transition-group": "2.6.0",
     "rollup-plugin-babel-minify": "^8.0.0",
+    "rss-parser": "^3.7.2",
```

Comment:
has umd support

Reply:
please try to exclude instead (this other rss parser does not provide access to the full html content)